src/ets_lru.erl

```diff
@@ -332,6 +339,8 @@ set_options(St, [{max_size, N} | Rest]) when is_integer(N), N >= 0 ->
     set_options(St#st{max_size=N}, Rest);
 set_options(St, [{max_lifetime, N} | Rest]) when is_integer(N), N >= 0 ->
     set_options(St#st{max_lifetime=N}, Rest);
+set_options(St, [{time_unit, T} | Rest]) when is_atom(T) ->
```

Comment:
Is there a reason to make it configurable?

Reply:
It requires relatively little overhead, and allows the user to choose something other than millisecond resolution if their use case requires it. However if you don't like it, I'm happy to remove it.